README.md

```diff
@@ -1,6 +1,28 @@
 # rly-network-mobile-sdk
 
-An SDK to make working with token economies as easy as credit cards
+## What is rly-network-mobile-sdk?
+Rly-network-mobile-sdk is a set of libraries, tools, and smart contracts that streamlines blockchain integration for developers to enable frictionless user onboarding and simplify on-chain transactions. 
```

Comment:
Probably worth adding in links to our open source smart contracts here. If we're going to mention them, probably should link to them.